README.md

```diff
@@ -43,7 +43,7 @@ It's modelled after `UIImagePickerController`, which makes it a breeze to use.
 
 ## Requirements
 
-- Swift 4.2
+- Swift 4.2+
```

Comment:
This makes me think we support Swift 5.0 out of the box and possibly future releases as well. We might want to stick with just Swift 4.2?

Reply:
My bad - I'll revert that.